test/testcafe/spec/ExpiredAuthenticatorPasswordView_spec.js

```diff
@@ -0,0 +1,90 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import AuthenticatorExpiredPasswordPageObject from '../framework/page-objects/AuthenticatorExpiredPasswordPageObject';
+import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
+import xhrAuthenticatorExpiredPassword from '../../../playground/mocks/data/idp/idx/authenticator-expired-password';
+import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
+
+const logger = RequestLogger(/challenge\/answer/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorExpiredPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrSuccess);
+
+fixture(`Authenticator Expired Password`);
+
+async function setup(t) {
+  const expiredPasswordPage = new AuthenticatorExpiredPasswordPageObject(t);
+  await expiredPasswordPage.navigateToPage();
+  return expiredPasswordPage;
+}
+
+test
```

Comment:
Should we add more tests to cover all policy cases?

Reply:
This addresses all possible policy cases for expired password as indicated by the mock. Do you know if I'm missing any? 

Also, when I address password will expire soon and admin initiated reset scenarios, the password requirements for those will be covered then

Reply:
> cover all policy cases

Good point and I'm leaning toward add unit test for `v1/FactorUtil/getPasswordComplexityDescriptionForHtmlList`

Reply:
Opened this - https://oktainc.atlassian.net/browse/OKTA-310415, will address separately